chromium_src/chrome/browser/flag_descriptions.cc

```diff
@@ -57,6 +57,10 @@ const char kBraveSyncDescription[] =
 const char kBraveIpfsName[] = "Enable IPFS";
 const char kBraveIpfsDescription[] =
     "Enable native support of IPFS.";
+const char kBraveRewardsVerboseLoggingName[] =
+    "Enable Brave Rewards verbose logging";
+const char kBraveRewardsVerboseLoggingDescription[] =
+    "Enables detailed logging of Brave Rewards system events to a log file";
```

Comment:
IMO this should have a warning that says this might log sensitive information.

Reply:
Agreed - I'll add some preliminary text and attach a screenshot.